src/main/java/be/seeseemelk/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -560,20 +545,62 @@ public void clearEvents()
 		events.clear();
 	}
 
+	@Override
+	public void registerEvents(Listener listener, Plugin plugin)
+	{
+		if (!plugin.isEnabled()) {
+			throw new IllegalPluginAccessException("Plugin attempted to register " + listener + " while not enabled");
+		}
+
+		for (Map.Entry<Class<? extends Event>, Set<RegisteredListener>> entry : plugin.getPluginLoader().createRegisteredListeners(listener, plugin).entrySet()) {
+			getEventListeners(getRegistrationClass(entry.getKey())).registerAll(entry.getValue());
+		}
+	}
+
 	@Override
 	public void registerEvent(Class<? extends Event> event, Listener listener, EventPriority priority,
 			EventExecutor executor, Plugin plugin)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		registerEvent(event,listener,priority,executor,plugin,false);
```

Comment:
Formatting. Please be consistent.

```suggestion
		registerEvent(event, listener, priority, executor, plugin, false);
```